resources/schemas/fixture.xsd

```diff
@@ -359,6 +359,8 @@ when changing this list -->
         <xs:enumeration value="3-pin"/>
         <xs:enumeration value="5-pin"/>
         <xs:enumeration value="3-pin and 5-pin"/>
+        <xs:enumeration value="3-pin IP65"/>
+        <xs:enumeration value="5-pin IP65"/>
```

Comment:
Technically, if you add entries here, you should add the same to the fixture editor, somewhere around here
https://github.com/mcallegari/qlcplus/blob/master/fixtureeditor/editphysical.ui#L213

not a big deal though